pkg/resources/dns/dns.go

```diff
@@ -50,10 +50,6 @@ var (
 // source: https://github.com/kubernetes/kubernetes/blob/vX.YY.0/cmd/kubeadm/app/constants/constants.go
 func GetCoreDNSImage(kubernetesVersion *semver.Version) string {
 	switch fmt.Sprintf("%d.%d", kubernetesVersion.Major(), kubernetesVersion.Minor()) {
-	case "1.19":
-		return "coredns/coredns:v1.7.0"
-	case "1.20":
-		return "coredns/coredns:v1.7.0"
```

Comment:
Nit: it might be safer to leave those cases in place and make default return `coredns/coredns:v1.8.4`. For example, if we for some reason forget to add a new case for a new Kubernetes version, it'll downgrade CoreDNS, which can be a bit problematic.